scripts/scripts/slack.groovy

```diff
@@ -0,0 +1,30 @@
+import jenkins.model.Jenkins
+import net.sf.json.JSONObject
```

Comment:
where does this come from: `net.sf.json.JSONObject`

Reply:
not sure but the codes mostly from the docs: https://plugins.jenkins.io/slack